README.md

```diff
@@ -22,6 +22,35 @@ A detailed tutorial is provided on our [website](https://federatedscope.io/).
 - [05-25-2022] Our paper [FederatedScope-GNN](https://arxiv.org/abs/2204.05562) has been accepted by KDD'2022!
 - [05-06-2022] We release FederatedScope v0.1.0! 
 
+## Code Structure
+```
+FederatedScope
+├── federatedscope
+│   ├── core           
+│   |   ├── worker               # Behaviors of participants (i.e., server and clients)
+│   |   ├── configs              # Customizable configurations
+│   |   ├── trainer              # Detail of local training
+│   |   ├── monitors             # The monitor module for logging and demonstrating  
+│   |   ├── aggregator.py        # Detail of federated aggregation
+│   |   ├── communication.py     # Implementation of communication among participants   
+│   |   ├── fed_runner.py        # The runner for building and running an FL course
+│   |   ├── ... ..
+│   ├── cv                       # Federated learning in CV        
+│   ├── nlp                      # Federated learning in NLP          
+│   ├── gfl                      # Graph federated learning          
+│   ├── autotune                 # Auto-tunning for federated learning         
+│   ├── main.py           
+│   ├── ... ...          
```

Comment:
maybe we should place the directory `contrib` at the same level with core/cv/nlp/gfl?

Reply:
done